src/Illuminate/Foundation/Vite.php

```diff
@@ -397,4 +419,53 @@ public function reactRefresh()
             )
         );
     }
+
+    /**
+     * Merge the given entry points.
+     *
+     * @param  array  $entryPoints
+     * @return $this
+     */
+    public function withEntryPoints(array $entryPoints)
+    {
+        $this->entryPoints = array_merge($this->entryPoints, $entryPoints);
+
+        return $this;
+    }
+
+    /**
+     * Set the hot file path attribute.
+     *
+     * @param  string
+     * @return $this
+     */
+    public function useHotFile(string $path)
+    {
+        $this->hotFilePath = Str::start($path, '/');
```

Comment:
I don't think we should enforce the path starting with a slash. I'd rather palm this off to the end user to have a prepared path - so they can do whatever funny stuff they wanna do. Relative, absolute, whatever.